src/rabbit.erl

```diff
@@ -30,6 +30,9 @@
 %% Boot steps.
 -export([maybe_insert_default_data/0, boot_delegate/0, recover/0]).
 
+%% for tests
+-export([validate_io_options/2]).
```

Comment:
This is a very generic name but we actually validate only 2 settings. I think this needs a more specific name.